keyboards/kbdfans/kbd67/readme.md

```diff
@@ -1,22 +1,28 @@
 # KBD67
 
-A 65% keyboard sold in four variants. 
-1. Rev1: Typical keyboard that had to be soldered together, supporting multiple layouts. The rev1 PCB is sold under the name "KBD65".  
-2. HotSwap: Released in late 2018, Hotswap single layout keyboard. 
-3. Rev2: Released in April/May 2019, the Rev2 also needs to be soldered together and supports multiple layouts. 
-4. MKII RGB: Released in September 2019. ARM powered hotswap board. 
+A 65% keyboard sold in several variants. 
+
+1. Rev1: Released in early 2018. Atmega32u4 soldered keyboard. Sold under the name "KBD65".  
+2. HotSwap: Released in late 2018. Atmega32U4 hotswap single layout keyboard. 
+3. Rev2: Released in April/May 2019. Atmega32u4 soldered keyboard.
+4. MKII: Released in September 2019, designed by ai03. Atmega32u4 soldered keyboard. ESD Protected
+5. MKII RGB V1: Released in September 2019. ARM hotswap board. 
+6. MKII RGB V2: Released in December 2019. Atmega32u4 hotswap board. ESD Protected.
```

Comment:
IMO this section should link to the appropriate revision folder, similar to how the Clueboard vendor readme works.

https://github.com/qmk/qmk_firmware/blob/8fa64568f01604bde0bb5eb1923e25483af03424/keyboards/clueboard/readme.md

Reply:
Not a bad idea